v0/destinations/appcues/data/AppcuesPageProfile.json

```diff
@@ -0,0 +1,12 @@
+{
+	"userId": "userId",
+	"_lastBrowserLanguage": "context.locale",
+	"_updatedAt": "sentAt",
```

Comment:
Ideally it should be `originalTimeStamp` or `timestamp`

Reply:
changed